x-pack/plugins/canvas/public/state/actions/transient.js

```diff
@@ -6,7 +6,6 @@
 
 import { createAction } from 'redux-actions';
 
-export const setCanUserWrite = createAction('setCanUserWrite');
 export const setFullscreen = createAction('setFullscreen');
 export const selectElement = createAction('selectElement');
 export const setFirstLoad = createAction('setFirstLoad');
```

Comment:
`setFirstLoad` should also be removed, since you removed the reducer and it's not used anymore.